src/middle/Middle.mli

```diff
@@ -1,27 +1,46 @@
-include module type of Mir
 open Core_kernel
+include module type of Mir
+include module type of Mir_pretty_printer
+module Validation : Validation_intf.Validation
```

Comment:
Do we still need this `_intf` pattern here? Sean seemed to be able to make the `include module type of Foo` syntax work even in cases where there was a `Foo.mli` file. Perhaps it can't be made to work at all with functors?

Reply:
I'll take a proper look at Sean's approach and see if it can be done.